frontend/package-lock.json

```diff
@@ -3009,6 +3009,19 @@
       "resolved": "https://registry.npmjs.org/@nuxtjs/gtm/-/gtm-2.2.3.tgz",
       "integrity": "sha512-YRvxCo2rpe61kR49LmepK0lCjrrb3hvb89X7/AO5IFPzPU6hxUDpi4V3rnahUTwHcDdYknVqtjAn8ZFMp+HE2Q=="
     },
+    "@nuxtjs/sentry": {
```

Comment:
Do we need this new dependency here? @johann-su

Reply:
Yes, will be enabled for production.

Reply:
@mskg Does that mean we don't need that for now?

Reply:
The change is not part of this PR